src/index.html

```diff
@@ -9,6 +9,13 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Snake</h1>
+  <div class="container">
+    <div class="block block-1">1</div>
+    <div class="block block-2">2</div>
+    <div class="block block-3">3</div>
+    <div class="block block-4">4</div>
+    <div class="block block-5">5</div>
+    <div class="block block-6">6</div>
```

Comment:
```suggestion
    <div class="block">1</div>
    <div class="block">2</div>
    <div class="block">3</div>
    <div class="block">4</div>
    <div class="block">5</div>
    <div class="block">6</div>
```
Стилизуй через `nth-child`, не надо добавлять отдельный класс